presto-jdbc/src/main/java/io/prestosql/jdbc/ConnectionProperties.java

```diff
@@ -318,7 +330,19 @@ public SslTrustStorePassword()
 
         public SslTrustStoreType()
         {
-            super("SSLTrustStoreType", NOT_REQUIRED, IF_TRUST_STORE, STRING_CONVERTER);
+            super("SSLTrustStoreType", NOT_REQUIRED, IF_TRUST_STORE.and(SslVerificationMode.IF_SSL_VERIFICATION_ENABLED), STRING_CONVERTER);
+        }
+    }
+
+    private static class SslVerification
+            extends AbstractConnectionProperty<SslVerificationMode>
+    {
+        private static final Predicate<Properties> IF_SSL_ENABLED =
```

Comment:
why this one is defined here, and `IF_SSL_VERIFICATION_ENABLED` is defined elsewhere?